### PR TITLE
Update styling of 'View Playlists' page

### PR DESCRIPTION
**Pull request by nickbircher**
Issue [87](https://github.com/ryanhu021/csc307-team-project/issues/87)

### Changes
- Add hover styling to playlist cards
- Adjust spacing
- Add hover styling to upload and delete icons

### Image References 
image

image

image

### CODE REVIEW

**Review by ktaschek — CHANGES_REQUESTED** [review]
Linting error in the frontend: build did not complete.

File: packages/frontend/src/component-styles/add-playlist-card.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.add-playlist-container {
```
Comment (ryanhu021) [inline packages/frontend/src/component-styles/add-playlist-card.css:1]: nit: we could try to migrate these css files to bootstrap classes or styled components if we have time

**Review by ryanhu021 — APPROVED** [review]
Looks good!

**Review by ktaschek — CHANGES_REQUESTED** [review]
New Playlist Card is not updated to have same sizing as the playlist cards

**Review by ktaschek — APPROVED** [review]
LGTM

**Pull request merged**